app/src/main/java/org/openimis/imispolicies/FeedbackList.java

```diff
@@ -442,7 +446,7 @@ public void run() {
         }
     }
     public void LoginDialogBox(final String page){
-
+        ca.CheckInternetAvailable();
```

Comment:
If there is no connectivity, it will not stop the function.

Reply:
Fixed